src/main/java/org/opentripplanner/api/resource/GraphPathToTripPlanConverter.java

```diff
@@ -406,6 +405,11 @@ private static Leg generateLeg(Graph graph, LegStateSplit legStateSplit, boolean
 
 
         leg.vehicleDescription = states.get(0).getCurrentVehicle();
+        if (leg.vehicleDescription != null) {
+            leg.activePackageIndex = states.get(states.size() - 1).getActivePackageIndex();
+        } else {
+            leg.activePackageIndex = -1;
```

Comment:
I think field `activePackageIndex` should be `Integer`, so we could set it to `null` here.